streams/test-utils/src/test/java/org/apache/kafka/streams/MockProcessorContextTest.java

```diff
@@ -243,10 +245,14 @@ public void process(final String key, final Long value) {
         };
 
         final MockProcessorContext context = new MockProcessorContext();
-        final KeyValueStore<String, Long> store = new InMemoryKeyValueStore<>("my-state", Serdes.String(), Serdes.Long());
-        context.register(store, null);
+        final StoreBuilder storeBuilder = Stores.keyValueStoreBuilder(
```

Comment:
I thought it would be the "correct" way to get a <String, Long> store but revisiting the issue maybe it's best to just use the wrapper store here as well and avoid the unnecessary overhead/ be consistent?

Reply:
Frankly, I am not sure if it makes a difference. Leave it up to you.